src/utils/selectors/QACert/TestSummary.js

```diff
@@ -708,6 +708,11 @@ export const getTableRowActionAriaLabel = (dataTableName, row, action) => {
     case "Test Extension Exemption": //unique ID is Extension or Exemption Code, i.e. col9
       result = `${action} for ${row.col9}`;
       break;
+    case "Linearity Test": 
+    case "Protocol Gas":
+    case "Linearity Injection":
```

Comment:
For the case `Linearity Injection`, using `row.col1` in the result leads to a dynamic value like a timestamp being included in the `aria-label`, e.g.,
`Edit for 2023-05-16 08:23 Linearity Injection`.

Including timestamps in `aria-labels` can reduce clarity for screen reader users and may introduce unnecessary confusion.